src/ittools/jira/jira_ext.py

```diff
@@ -160,29 +162,29 @@ def fix_versions(self) -> List[str]:
         return list(map(lambda version: version["name"], versions))
 
     def add_fix_version(self, new_fix_version) -> None:
-        self.raw_issue.add_field_value("fixVersions", {"name": new_fix_version})
+        self.raw_issue.add_field_value("fixVersions", new_fix_version)
```

Comment:
This causes a regression:

`response text = {"errorMessages":[],"errors":{"fixVersions":"Could not find valid 'id' or 'name' in version object."}}`

Reply:
Thanks, should be resolved now.

Reply:
It still shows the same error message, I'm afraid. The documentation appears to be completely wrong in this instance.

Reply:
Would you accept bypassing this broken function?:
 ```py
      self.raw_issue.update(update={"fixVersions": [{"add": {"name": new_fix_version}}]})
```

Reply:
Yep, that works